docs/tutorials/tabular_prediction/tabular-gpu.md

```diff
@@ -41,19 +41,47 @@ hyperparameters
 
 ### Text models
 
-Text model preset to use can be set via:
+Text model's configuration can be set via:
```

Comment:
[!!!] Potential bug:

[Logs](http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1547/6/tutorials/tabular_prediction/tabular-multimodal.html) shows TextPredictor getting 0.3 val score in training logs, but in Tabular leaderboard / val score it only has 0.29 (worse).

Is this because of the weight averaging at end of fit? @sxjscience I believe we talked about this before, but I still recommend we check val score of weight averaged model and use non-weight averaged version if weighted averaging makes val score worse.

Reply:
@zhiqiangdon Need to double check the cause.

Reply:
Could also go further by evaluating multiple weight average checkpoints: top-2, top-3, top-2-weight-by-val-score, top-3-weight-by-val-score, even HPO on weights of top-N if the process to do so is fast.

Reply:
The risk is that it may give us worse performance in the test set (since we are tuning it on the validation set). The purpose of weight averaging is to ensure that the model generalize better. Nevertheless, we can add a flag about this behavior and verify it with benchmark.